src/app/snackbar/snackbar.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+import { AppUpdaterService } from '../services/app-updater.service';
+
+@Component({
+  selector: 'mav-snackbar',
+  templateUrl: './snackbar.component.html',
+  styleUrls: ['./snackbar.component.scss']
+})
+export class SnackbarComponent {
```

Comment:
Why is this generically called Snackbar Component? Shouldn't it be something like UpdateNotificationComponent or something along those lines?